java/flight/flight-sql-jdbc-core/src/main/java/org/apache/arrow/driver/jdbc/ArrowFlightJdbcFlightStreamResultSet.java

```diff
@@ -73,6 +86,7 @@ public final class ArrowFlightJdbcFlightStreamResultSet
                                        final Meta.Frame firstFrame) throws SQLException {
     super(null, state, signature, resultSetMetaData, timeZone, firstFrame);
     this.connection = connection;
+    initializeVectorSchemaRootsQueue();
   }
```

Comment:
Would it be possible to have both constructors delegate to a single constructor, then inline initializeVectorSchemaRootsQueue into that constructor?